webpack.config.ts

```diff
@@ -186,35 +192,18 @@ const plugins: webpack.WebpackPluginInstance[] = [];
 plugins.push(new CopyPlugin({
   patterns: [
     {
-      from: path.resolve(__dirname, 'node_modules/react/umd'),
-      to: path.resolve(__dirname, 'htdocs/vendor/js/react'),
+      from: `node_modules/react/umd/${
+        isDev ? 'react.development.js' : 'react.production.min.js'
+      }`,
+      to: 'htdocs/vendor/js/react',
       force: true,
-      globOptions: {
-        ignore: ['react.profiling.min.js'],
-      },
```

Comment:
Why was this removed / what is the implication of its removal?

Reply:
TBH I am pretty sure it was dead code. Before, all the whole `umd` directory was marked for copy, but the `filter` function ignored all files but two (`react.development.js` and `react.production.min.js`). Now, only these two files are copied (or only the one needed depending on whether this is a dev or prod build more precisely), so there is no need to ignore the other files of the directory anymore.